### PR TITLE
New autolathe category for kitchen+bar items

### PR DESCRIPTION
**Pull request by erwgd**
The autolathe has a new category, Dinnerware. The kitchen knife has been moved to this category. Forks, bowls, drinking glasses and butcher's cleavers can now be made from this category, cleavers being available if the autolathe is hacked.
Materials for the newly autolatheable items have been defined: 500 glass for a drinking glass or a bowl, same as beakers; 80 metal for a fork; 150 glass for a shot and 1500 metal for a shaker. Materials for a butcher's cleaver have been overrided to be 18000 metal.

This PR will add a way to make more bowls, making bug #16519 less of a problem.

:cl: Erwgd
rscadd: Forks, bowls, drinking glasses, shot glasses and shakers can now be made and recycled in the autolathe! Access the designs in the new 'Dinnerware' category.
tweak: Kitchen knives have been moved to the Dinnerware category of the autolathe.
rscadd: Butcher's cleavers can now be made in a hacked autolathe.
/:cl:

### CODE REVIEW

**Comment by octareenroon91** [comment]
Why biogen? Just to keep it balanced with the autolathe?

**Comment by erwgd** [comment]
The biogenerator is generally more accessible to the cook as he (with extended access) or the botanist have access to it. Bowls are a little more likely to be in demand than the other dinnerware as they are required for some dishes, so making them more accessible is important. Bowls made of plant fibers do exist, so it makes enough sense for us to be able to make them in the biogen.

I did make bowls available from the autolathe (requiring 500 glass) as well as the biogen, but it didn't sit right with me to allow bowls from the biogen to be recycled into glass for the autolathe. I did code a glass-free bowl for the biogenerator, but soups and salads when eaten would still produce normal glass bowls.

File: code/modules/food&drinks/food/customizables.dm [hunk 1]
```diff
@@ -300,6 +300,7 @@
 	icon = 'icons/obj/food/soupsalad.dmi'
 	icon_state = "bowl"
 	flags = OPENCONTAINER
+	materials = list()
```
Comment (Fox-McCloud) [inline code/modules/food&drinks/food/customizables.dm]: glass bowls...without glass.

Comment (PKPenguin321) [inline code/modules/food&drinks/food/customizables.dm]: ours aren't glass they're like a weird clay

Comment (Fox-McCloud) [inline code/modules/food&drinks/food/customizables.dm]: Glass clay bowls that aren't glass, even better!

Hheheh, thanks for clearing it up.

**Comment by lordpidey** [comment]
Could you also add shotglasses to the autolathe?

**Comment by Cheridan** [comment]
I'd really like for the biogenerator to be limited to things that are actually bio.

**Comment by erwgd** [comment]
@Cheridan : OK, I removed the bowl code from biogenerator.dm. Bowls will instead be made of 500 glass, and can be made from the autolathe.

**Pull request merged**